### PR TITLE
Initial structure

### PR DESCRIPTION
**Pull request by alexcoman**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+setup(
+    name="instacli",
+    version="0.1",
+    description="Instagram command line application",
+    long_description=open("README.md").read(),
+    author="Teona Rusu, Matei Micu, Alexandru Coman",
+    url="https://github.com/micumatei/InstaCLI",
+    packages=["instacli"],
+    scripts=["scripts/instacli"],
```
Comment (TeonaRusu) [inline setup.py]: Script-ul se numeste `insta-cli`.

**Comment by alexcoman** [comment]
@micumatei @TeonaRusu review please.

**Comment by micumatei** [comment]
LGTM

**Comment by TeonaRusu** [comment]
GG

**Pull request merged**